pkg/networkservice/core/next/next_client_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 Doc.ai, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package next
```

Comment:
Filename `next_client_test.go` is excessive,  `client_test.go` should be enough

Reply:
thanks, fixed.